docs/source/bentocloud/how-tos/manage-models-and-bentos.rst

```diff
@@ -1,97 +1,54 @@
-===========================
+========================
 Manage Models and Bentos
-===========================
+========================
 
-In this guide, we'll walk through the process of managing your Models and Bentos
-effectively in BentoML. This includes pulling and pushing Models and Bentos,
-dealing with new versions, and viewing their status.
+BentoML provides both a local Model Store and a Bento Store to host your models and Bentos, respectively. Once you've pushed these entities to BentoCloud,
+they are stored in remote repositories. This setup helps teams manage them in a unified way, making it easier to share and collaborate with team members.
 
-Prerequisite
-============
+This document explains how to pull and push models and Bentos.
 
-Ensure you have BentoML installed and are properly logged into your BentoCloud
-account. If you're not yet logged in, use the following command:
+Prerequisites
+=============
 
-.. code-block:: bash
+- You have :doc:`installed BentoML </quickstarts/install-bentoml>`.
+- You have logged in to BentoCloud. See :doc:`Manage Access Tokens <manage-access-token>` to learn more.
 
-   bentoml cloud login --api-token <your-api-token> --endpoint <your-bentocloud-endpoint>
-
-.. note::
-
-   Refer to :doc:`Manage Access Tokens <manage-access-token>`
-   on how to obtain your access token.
-
-Local and Remote Repositories
-=============================
-
-Models and Bentos are initially created and stored locally.
-BentoCloud provides a remote repository feature, similar to Git,
-which enables you to push these local assets to a remote repository
-for efficient management and team collaboration.
-
-Pull and Push Models
+Pull and push models
```

Comment:
What is our capitalization strategies for titles? I thought the common ones are to capitalize most verb, nouns, adjectives except stopwords. i.e.

`Pull and Push Models`

Reply:
I am thinking about using sentence case for our titles, namely capitalizing just the first letter of the first word in the title (but obviously special terms like BentoML, BentoCloud, and Bentos need certain letters capitalized). The reason for this is that this rule is very simple. Title case rules are often not unified. You need to explain to people what word types need to be capitalized and this may confuse some. In terms of open source contribution, sentence case is actually a more easy choice. BTW, I will change the capitalization for all docs going forward to make them consistent.